features/beforeunload.yml.dist

```diff
@@ -2,32 +2,15 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: high
-  baseline_low_date: 2015-07-29
-  baseline_high_date: 2018-01-29
+  baseline: false
```

Comment:
This is a Baseline status regression, due to this BCD change: https://github.com/mdn/browser-compat-data/pull/26054
Apparently, `beforeunload` doesn't work in Safari for iOS.